packages/meditrak-server/src/apiV2/import/importStriveLabResults/importStriveLabResults.js

```diff
@@ -29,7 +29,15 @@ const createWorkBookParser = () => {
 const createImporter = models => {
   const constructResponseExtractor = () => async input => {
     const entityCode = input[ENTITY_CODE_KEY];
-    const { id: entityId } = await models.entity.findOne({ code: entityCode });
+    const getEntityId = async () => {
+      try {
+        const { id } = await models.entity.findOne({ code: entityCode });
+        return await id;
```

Comment:
No need to `await` the id here :-) 
```suggestion
        return id;
```

Reply:
Thanks @edmofro . Yes I wasn't sure about whether await was required inside the try/catch promise structure.